ccloud/docs/beginner-cloud.rst

```diff
@@ -0,0 +1,902 @@
+
+   .. code-block:: text
+
+         ServiceAccountId | Permission | Operation | Resource |  Name   |  Type
+         +------------------+------------+-----------+----------+---------+---------+
+         User:104349      | ALLOW      | READ      | GROUP    | connect | LITERAL
+
+#. Verify the ACLs were configured by running the following command:
+
+   .. code-block:: bash
+
+      ccloud kafka acl list --service-account 104349
+
+   .. code-block:: text
+
+         ServiceAccountId | Permission | Operation | Resource |  Name   |  Type
+       +------------------+------------+-----------+----------+---------+---------+
+         User:104349      | ALLOW      | WRITE     | TOPIC    | *       | LITERAL
+         User:104349      | ALLOW      | CREATE    | TOPIC    | *       | LITERAL
+         User:104349      | ALLOW      | READ      | TOPIC    | *       | LITERAL
+         User:104349      | ALLOW      | READ      | GROUP    | connect | LITERAL
+
+#. Generate environment variables with |ccloud| connection information for
+   |kconnect| to use:
+
+   .. code-block:: text
+
+      ../../ccloud/ccloud-generate-cp-configs.sh /tmp/client.config &>/dev/null
+      source delta_configs/env.delta
+
+# Run a |kconnect| container with the kafka-connect-datagen plugin:
+
+  .. code-block:: bash
+
+     docker-compose up -d
+
+  You should see the following output:
+
+  .. code-block:: text
+
+      Creating connect-cloud ... done
+      Waiting up to 60 seconds for Docker container for connect to be up
+      ............
+
+#. Post the configuration for the kafka-connect-datagen connector that produces
+   pageviews data to |ccloud| topic ``demo-topic-3``:
+
+   .. code-block:: bash
+
+      curl --silent --output /dev/null -X POST -H "Content-Type: application/json" --data
+
+   .. code-block:: text
+
+      "{
+      "name": "datagen-demo-topic-3",
+      "config": {
+         "connector.class": "io.confluent.kafka.connect.datagen.DatagenConnector",
+         "kafka.topic": "demo-topic-3",
+         "quickstart": "pageviews",
+         "key.converter": "org.apache.kafka.connect.storage.StringConverter",
+         "value.converter": "org.apache.kafka.connect.json.JsonConverter",
+         "value.converter.schemas.enable": "false",
+         "max.interval": 5000,
+         "iterations": 1000,
+         "tasks.max": "1"
+      }
+      }" http://localhost:8083/connectors
+
+
+#. Wait 20 seconds for kafka-connect-datagen to start producing messages.
+
+#. Run the following command to verify connector is running:
+
+   .. code-block:: bash
+
+      curl --silent http://localhost:8083/connectors/datagen-demo-topic-3/status
+
+   .. code-block:: text
+
+      {"name":"datagen-demo-topic-3","connector":
+      {"state":"RUNNING","worker_id":"connect:8083"},"tasks":
+      [{"id":0,"state":"RUNNING","worker_id":"connect:8083"}],"type":"source"}
+
+#. Create ACLs for the consumer using a wildcard by running the following
```

Comment:
This should be moved to the next section

Reply:
Updated